extensions/amp-story/1.0/amp-story-reaction-quiz.css

```diff
@@ -0,0 +1,447 @@
+/**
```

Comment:
Similar to amp-story-quiz.css (deleted), didn't change any styles or rules besides naming and some glitches.

Reply:
Where does the 13 lines diff come from?

Reply:
Two rules were overriding the icons colors (checks and crosses) and making them invisible on post-selection, so I took the rules away.